src/matter/interaction/InteractionMessages.ts

```diff
@@ -11,6 +11,31 @@ import { AnyT, ArrayT, BooleanT, Field, ObjectT, OptionalField, Template, Unsign
 export const enum StatusCode {
     Success = 0x00,
     Failure = 0x01,
+    InvalidSubscription = 0x7d,
```

Comment:
Can you add a ref to the spec section where does values are defined?

Reply:
done